### ISSUE TITLE
[NG-Alerting] Microsoft Teams template missing new-lines before Annotations & Silence (cosmetic issue)

### ISSUE THREAD

**Issue by diveflo** [grafana/grafana#36049]
**What happened**:
I setup the ng-alerting with Microsoft Teams webhook as a contact point. The alerts are sent as expected and contain the correct information. This is just a cosmetic issue.
The default template for the Microsoft Teams notification seems to be missing two new-lines: one after the last **Label** and before **Annotations** and one after the last **Annotation** and before **Silence**

In the example image,  severity is my last label

![image](https://user-images.githubusercontent.com/11045082/123009234-9e6a2180-d3bc-11eb-8231-92f646cf55ff.png)

**What you expected to happen**:
I would expect two bullet point lists: Labels, Annotations and (after a new line): the silence link

**Environment**:
- Grafana version: 8.0.3
- Data source type & version: InfluxDB 1.8
- OS Grafana is installed on: Docker
- User OS & Browser: Windows 10/Chromium
- Grafana plugins: n/a

**Comment by dsiperek-vendavo** [comment]
<img width="869" alt="Screen Shot 2021-06-23 at 10 57 54 AM" src="https://user-images.githubusercontent.com/70541548/123146433-b2407280-d412-11eb-9d41-466e40f8a45c.png">
Another example of same issue.

**Comment by codesome** [comment]
There is actually a `\n` between those, but looks like Teams requires a double newline?

In the optional message field of Teams settings, could you try this template and see if it is adding a newline as expected?
```
{{ range .Alerts.Firing }}
Labels:
{{ range .Labels.SortedPairs }} - {{ .Name }} = {{ .Value }}
{{ end }}
Annotations:
{{ range .Annotations.SortedPairs }} - {{ .Name }} = {{ .Value }}
{{ end }}
{{ if gt (len .GeneratorURL) 0 }}Source: {{ .GeneratorURL }}

{{ end }}{{ if gt (len .SilenceURL) 0 }}Silence: {{ .SilenceURL }}

{{ end }}{{ if gt (len .DashboardURL) 0 }}Dashboard: {{ .DashboardURL }}

{{ end }}{{ if gt (len .PanelURL) 0 }}Panel: {{ .PanelURL }}

{{ end }}{{ end }}
```

**Comment by diveflo** [comment]
Yeah, like this it looks good:)

![image](https://user-images.githubusercontent.com/11045082/123255994-bcd13980-d4f0-11eb-8e27-ad7f9abee353.png)

While we're already talking about it: might it be possible to add one more new line between at the end?
Currently, if you have multiple firing alerts, the next one is described without any break in the notification:
![image](https://user-images.githubusercontent.com/11045082/123259381-af1db300-d4f4-11eb-8b29-7d51a30d9ac6.png)

**Comment by codesome** [comment]
Thanks for testing it out, will update the template for Teams (with the additional newline at the end)

**Comment by codesome** [comment]
Fix is in https://github.com/grafana/grafana/pull/36126/. @diveflo will you be able to test it out to see if it is working fine? (will merge otherwise too)

**Comment by diveflo** [comment]
Awesome - thanks!
Let me quickly build a docker image of the PR and test it:)

**Comment by codesome** [comment]
I have merged it so that it can make it to 8.0.4

**Comment by diveflo** [comment]
Looks good:) Thank u
Sorry the build took a bit :D
